doc/book/admin/access_control.rst

```diff
@@ -1,377 +1,1069 @@
+
+..  NOTE::
+
+    Information about users who gave the specified privileges is stored in the :ref:`_priv <box_space-priv>` space.
 
-.. code-block:: lua
 
-   -- This example will work for a user with many privileges, such as 'admin'
-   -- or a user with the pre-defined 'super' role
-   -- Create space T with a primary index
-   box.schema.space.create('T')
-   box.space.T:create_index('primary', {})
-   -- Create the user U1 so that later the current user can be changed to U1
-   box.schema.user.create('U1')
-   -- Create two roles, R1 and R2
-   box.schema.role.create('R1')
-   box.schema.role.create('R2')
-   -- Grant role R2 to role R1 and role R1 to user U1 (order doesn't matter)
-   -- There are two ways to grant a role; here the shorter way is used
-   box.schema.role.grant('R1', 'R2')
-   box.schema.user.grant('U1', 'R1')
-   -- Grant read/write privileges for space T to role R2
-   -- (but not to role R1, and not to user U1)
-   box.schema.role.grant('R2', 'read,write', 'space', 'T')
-   -- Change the current user to user U1
-   box.session.su('U1')
-   -- An insertion to space T will now succeed because (due to nested roles)
-   -- user U1 has write privilege on space T
-   box.space.T:insert{1}
-
-More details are to be found in
-:doc:`/reference/reference_lua/box_schema/user_grant` and
-:doc:`/reference/reference_lua/box_schema/role_grant` in
-the built-in modules reference.
 
 .. _authentication-sessions:
 
---------------------------------------------------------------------------------
-Sessions and security
---------------------------------------------------------------------------------
+Sessions
+~~~~~~~~
 
-A **session** is the state of a connection to Tarantool. It contains:
+A session is the state of a connection to Tarantool.
+The session contains:
 
-* An integer ID identifying the connection,
-* the :ref:`current user <authentication-users>` associated with the connection,
-* text description of the connected peer, and
-* session local state, such as Lua variables and functions.
+*   An integer ID identifying the connection.
+*   The current :ref:`user <authentication-users>` associated with the connection.
+*   The text description of the connected peer.
+*   A session's local state, such as Lua variables and functions.
 
 In Tarantool, a single session can execute multiple concurrent transactions.
 Each transaction is identified by a unique integer ID, which can be queried
-at start of the transaction using :doc:`/reference/reference_lua/box_session/sync`.
+at the start of the transaction using :doc:`/reference/reference_lua/box_session/sync`.
+
+..  NOTE::
+
+    To track all connects and disconnects, you can use :ref:`connection and authentication triggers <triggers>`.
+
+
+
+
+.. _access_control_users:
+
+Managing users
+--------------
+
+.. _access_control_user_creating:
+
+Creating a user
```

Comment:
An idea for this an similar task-oriented subsections: mention a privilege that needed for doing this.

```
Creating a user

Required privilege: `write` on the `_users` space.

To create a new user ...
```